Library/Homebrew/cask/audit.rb

```diff
@@ -324,6 +324,7 @@ def check_hosting_with_appcast
 
     def check_desc
       return if cask.desc.present?
+      return if cask.tap == "homebrew/cask-fonts"
```

Comment:
@vitorgalvao One minor request: could you add a comment explaining why this is the case?

Reply:
Done.